torch/_utils.py

```diff
@@ -962,3 +962,111 @@ def fire_callbacks(self, *args: P.args, **kwargs: P.kwargs) -> None:
+    "SimpleDialog": "tkinter.simpledialog",
+    "DocXMLRPCServer": "xmlrpc.server",
+    "SimpleHTTPServer": "http.server",
+    "CGIHTTPServer": "http.server",
+    # For compatibility with broken pickles saved in old Python 3 versions
+    "UserDict": "collections",
+    "UserList": "collections",
+    "UserString": "collections",
+    "whichdb": "dbm",
+    "StringIO": "io",
+    "cStringIO": "io",
+}
+
+
+# This contains rename rules that are easy to handle.  We ignore the more
+# complex stuff (e.g. mapping the names in the urllib and types modules).
+# These rules should be run before import names are fixed.
+NAME_MAPPING = {
+    ("__builtin__", "xrange"): ("builtins", "range"),
+    ("__builtin__", "reduce"): ("functools", "reduce"),
+    ("__builtin__", "intern"): ("sys", "intern"),
+    ("__builtin__", "unichr"): ("builtins", "chr"),
+    ("__builtin__", "unicode"): ("builtins", "str"),
+    ("__builtin__", "long"): ("builtins", "int"),
+    ("itertools", "izip"): ("builtins", "zip"),
+    ("itertools", "imap"): ("builtins", "map"),
+    ("itertools", "ifilter"): ("builtins", "filter"),
+    ("itertools", "ifilterfalse"): ("itertools", "filterfalse"),
+    ("itertools", "izip_longest"): ("itertools", "zip_longest"),
+    ("UserDict", "IterableUserDict"): ("collections", "UserDict"),
+    ("UserList", "UserList"): ("collections", "UserList"),
+    ("UserString", "UserString"): ("collections", "UserString"),
+    ("whichdb", "whichdb"): ("dbm", "whichdb"),
+    ("_socket", "fromfd"): ("socket", "fromfd"),
+    ("_multiprocessing", "Connection"): ("multiprocessing.connection", "Connection"),
+    ("multiprocessing.process", "Process"): ("multiprocessing.context", "Process"),
+    ("multiprocessing.forking", "Popen"): ("multiprocessing.popen_fork", "Popen"),
+    ("urllib", "ContentTooShortError"): ("urllib.error", "ContentTooShortError"),
+    ("urllib", "getproxies"): ("urllib.request", "getproxies"),
+    ("urllib", "pathname2url"): ("urllib.request", "pathname2url"),
+    ("urllib", "quote_plus"): ("urllib.parse", "quote_plus"),
+    ("urllib", "quote"): ("urllib.parse", "quote"),
+    ("urllib", "unquote_plus"): ("urllib.parse", "unquote_plus"),
+    ("urllib", "unquote"): ("urllib.parse", "unquote"),
+    ("urllib", "url2pathname"): ("urllib.request", "url2pathname"),
+    ("urllib", "urlcleanup"): ("urllib.request", "urlcleanup"),
+    ("urllib", "urlencode"): ("urllib.parse", "urlencode"),
+    ("urllib", "urlopen"): ("urllib.request", "urlopen"),
+    ("urllib", "urlretrieve"): ("urllib.request", "urlretrieve"),
+    ("urllib2", "HTTPError"): ("urllib.error", "HTTPError"),
+    ("urllib2", "URLError"): ("urllib.error", "URLError"),
+    # Non-mutual mappings.
+    ("__builtin__", "basestring"): ("builtins", "str"),
+    ("exceptions", "StandardError"): ("builtins", "Exception"),
+    ("UserDict", "UserDict"): ("collections", "UserDict"),
+    ("socket", "_socketobject"): ("socket", "SocketType"),
```

Comment:
I don't think we care about safely loading those ever, especially if they were serialized using Python-2, so let's delete
```suggestion
```